src/EA.Weee.Core.Tests.Unit/AatfReturn/AddressDataTests.cs

```diff
@@ -13,9 +13,9 @@ public class AddressDataTests
         [InlineData("Address1")]
```

Comment:
Can we rename this file and tests to Site address tests and also create the same tests for the Aatf address